packages/api/config/tables.yml

```diff
@@ -2,6 +2,19 @@ CollectionsTable:
   read: 5
   write: 1
   pointInTime: '{{enablePointInTime}}'
+  enableAutoScaling: '{{tables.collections.enableAutoScaling}}'
```

Comment:
It seems like we're specifying all this info in 2 places and I don't understand the value of doing so

Reply:
An application owner can have control over which tables use auto-scaling by configuring these values in their deployment's `app/config.yml` file

Reply:
The comment wasn't just for this particular line but for 5-17.  All of those lines are just referencing variables from more or less one place (could easily be made into 1 place by adding indirection to `iams.scalingRoleArn`).

I don't understand why all the tables each take 13 lines to say "Use the configuration values over there...".  Could we just provide a way to say that?

Reply:
It probably seems nitpicky, but more lines = more errors and more clutter that makes it harder to spot what's actually going on.